.gitmodules

```diff
@@ -1,9 +1,9 @@
-[submodule "adsy-roles"]
-	path = adsy-roles
-	url = git@git.adfinis-sygroup.ch:ad-sy/ansible-roles.src.git
+[submodule "adfinis-roles"]
+	path = adfinis-roles
+	url = https://github.com/adfinis-sygroup/ansible-roles.git
 [submodule "doc/adsy-sphinx-template.src"]
 	path = doc/adsy-sphinx-template.src
```

Comment:
Maybe align the submodule path and path variables by removing ".src".

Reply:
Removed the `.src` and `adsy`